src/query/sql/src/planner/binder/project.rs

```diff
@@ -150,27 +153,150 @@ impl<'a> Binder {
         let mut output = SelectList::<'a>::default();
         for select_target in select_list {
             match select_target {
-                SelectTarget::QualifiedName(names) => {
+                SelectTarget::QualifiedName {
+                    qualified: names,
+                    exclude,
+                } => {
                     // Handle qualified name as select target
+                    let mut exclude_cols: HashSet<String> = HashSet::new();
+                    if let Some(exclude) = exclude {
+                        match exclude {
+                            ExcludeCol::Col(col) => {
+                                exclude_cols.insert(col.name.clone());
+                            }
+                            ExcludeCol::Cols(cols) => {
```

Comment:
Why `ExcludeCol::Col` and `ExcludeCol::Cols` are concomitant?  

When `Cols` has only one column, isn't it `Col`?

Reply:
I think we need judge `* exclude (id, id) ` the col is duplicate.

Reply:
Seems  `ExcludeCol::Cols(cols)` can cover `ExcludeCol::Col(col) `, so I recommend to keep `Cols`.

An extra judge for single item vec is trivial, but the code could be cleaner.

Reply:
> Seems `ExcludeCol::Cols(cols)` can cover `ExcludeCol::Col(col) `, so I recommend to keep `Cols`.
> 
> An extra judge for single item vec is trivial, but the code could be cleaner.

Now it already change to

```rust
QualifiedName {
  qualified: QualifiedName<'a>,
  exclude: Option<Vec<Identifier<'a>>>,
},
```

So we can drop struct ExcludeCol